cordova-sdk/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -158,13 +162,63 @@ enum InAppConsentStrategy: String {
             }
         }
 
+        if let enableDelayedInitialization = configValues[delayedInitializationEnable],
```

Comment:
silly github doesnt lt commenting on unchanged line

here `if (configValues[optimoveMobileCredentialsKey] == nil) {`

should have same logic as in android. Currently if there is no optimoveMobileCredentialsKey (e.g. for any delayed init), we wont do optimobile initialisation below